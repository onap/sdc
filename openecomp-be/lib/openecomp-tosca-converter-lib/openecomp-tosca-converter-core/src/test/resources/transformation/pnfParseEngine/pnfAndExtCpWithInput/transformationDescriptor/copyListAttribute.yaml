- transformation-for: nodeTemplate
  query:
    type: tosca.nodes.nfv.PnfExtCp
  conversions:
    - query: #attribute query: this attribute does not exists
        type1:
      to-name: type
      to-value:
        strategy: replace
        from: tosca.nodes.nfv.PnfExtCp
        to: org.openecomp.resource.cp.v2.extCP
    - query: #attribute query: this attribute exists
        properties:
          layer_protocols:
      to-name: layer_protocols_copy
      to-value:
        strategy: copy