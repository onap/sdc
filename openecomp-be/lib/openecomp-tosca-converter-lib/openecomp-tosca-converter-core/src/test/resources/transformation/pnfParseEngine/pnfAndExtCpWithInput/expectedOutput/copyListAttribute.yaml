topology_template:
  node_templates:
    pnfExtCp_1:
      properties:
        layer_protocols_copy:
          - ipv4
          - ipv6
          - otherProtocol